frontend/src/libs/types/server-error-handling.type.ts

```diff
@@ -0,0 +1,8 @@
+import { type HttpError } from '../packages/http/http.js';
+
+type ServerErrorHandling = {
+  error: HttpError | undefined;
```

Comment:
```suggestion
  error: HttpError | null;
```
undefined is more for optional arguments, if property is required it's better to use null